### PR TITLE
Dict: Add *sourc*

### PR DESCRIPTION
**Pull request by sebweb3r**
inspired by #1840

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -24042,6 +24042,8 @@ resoucing->resourcing
 resoultion->resolution
 resoultions->resolutions
 resourceype->resourcetype
+resourcs->resources, resource,
+resourcse->resources, resource,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:24046]: Add resourcd typo too.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -25817,6 +25819,8 @@ souldn't->shouldn't
 soundard->soundcard
 sountrack->soundtrack
 sourcedrectory->sourcedirectory
+sourcs->sources, source,
+sourcse->sources, source,
```
Comment (peternewman) [inline codespell_lib/data/dictionary.txt:25823]: Add sourcd typo

**Review by peternewman — CHANGES_REQUESTED** [review]
A few more comments

**Comment by sebweb3r** [comment]
push @peternewman

**Review by peternewman — APPROVED** [review]
LGTM thanks @sebweb3r

**Pull request merged**